README.md

```diff
@@ -256,7 +264,8 @@ nginx-ingress-controller-0f7r6   1/1       Running   0          1h
 #### List the services and endpoints
 
 ```
-$ kubectl get svc,ep
+
```

Comment:
We can not have empty lines between a tick fence and the text or it will not render correctly on jujucharms.com